portalwire/portal_protocol.go

```diff
@@ -294,6 +301,7 @@ func NewPortalProtocol(config *PortalProtocolConfig, protocolId ProtocolId, priv
 		NAT:               config.NAT,
 		clock:             config.clock,
 		Utp:               utp,
+		currentVersions:   key,
```

Comment:
the key is actual version?

Reply:
I still think it should be renamed, as it just assign the key to currentVersions